Dockerfile.in

```diff
@@ -4,7 +4,8 @@ FROM {{base}}
 # pre-package installation commands work + support https mirrors
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
-    ca-certificates curl gnupg apt-transport-https
+        ca-certificates curl gnupg apt-transport-https \
```

Comment:
I think it's good practice to add an `apt-get clean` here as well? E.g. https://github.com/ocf/ocfweb/blob/2f5766457e99355675cbe0bd2d0ebd4ed91b63ac/Dockerfile#L24